package/gluon-config-mode-contact-info/luasrc/lib/gluon/config-mode/wizard/0500-contact-info.lua

```diff
@@ -19,6 +19,20 @@ function M.section(form)
   o.rmempty = not ((site.config_mode or {}).owner or {}).obligatory
   o.datatype = "string"
   o.description = i18n.translate("e.g. E-mail or phone number")
+  o.validate = function(self, val)
+    if val == nil then
+      self.section.error = {
+        [1] = {
+          i18n.translate(
+          'You didn\'t provide any contact information! If you really want to run your '
+          .. 'node anonymously, you can enter a blank here. Please tell us an alternative '
+          .. 'how we could contact you in case there is anything wrong with your node.'
```

Comment:
I'm not sure if I like this text. The exclamation mark seems a bit over the top; I'm not sure how common the word "blank" is, and recommending a way around our own restrictions is somewhat weird.

"alternative how we could contact you" -> "alternative way to contact you"

Also, using `"` quotation marks instead of `'` would be better here, as you wouldn't need to escape the `'` in the text. In contrast to other languages, `"` and `'` strings aren't interpreted differently in Lua, the only difference is which quotationn marks need to be escaped in the text.